profile_bluesky/startup/instrument/devices/others.py

```diff
@@ -0,0 +1,33 @@
+'''
+Other motor/counters
+'''
+
+__all__ = [
+    'uptab','kbic','xbpm'
+    ]
+
+from ..session_logs import logger
+logger.info(__file__)
+
+from ophyd import Component, Device, EpicsMotor
+
+class UpTable(Device):
+    y = Component(EpicsMotor, 'm10', labels=('motor','uptable'))  # Uptable Y
+    x = Component(EpicsMotor, 'm9', labels=('motor','uptable'))  # Uptable Y
+
+uptab = UpTable('4iddx:',name='uptable')
+
+class KBIC(Device):
```

Comment:
consider putting in (new file) `kbic.py`

Makes these easier to find when editing later.